runtime/syntax/python.vim

```diff
@@ -77,13 +77,14 @@ endif
 "
 " The list can be checked using:
 "
-" python3 -c 'import keyword, pprint; pprint.pprint(keyword.kwlist, compact=True)'
+" python3 -c 'import keyword, pprint; pprint.pprint(keyword.kwlist, compact=True);\
+"" ;  pprint.pprint(keyword.softkwlist, compact=True)'
 "
 syn keyword pythonStatement	False None True
 syn keyword pythonStatement	as assert break continue del global
 syn keyword pythonStatement	lambda nonlocal pass return with yield
 syn keyword pythonStatement	class def nextgroup=pythonFunction skipwhite
-syn keyword pythonConditional	elif else if
+syn keyword pythonConditional	elif else if match case
```

Comment:
If we follow the output for keywords it does not contain `match` and `case`.
That's why we have to add soft keyword list.
I'm inclined to put them on a separate line for that reason:
```
syn keyword pythonConditional    match case
```
Can we make that small adjustments?

Reply:
Of course, I'll do that

Reply:
Done in new commit (force push)